packages/dom/src/core/plugins/feedback/Feedback.ts

```diff
@@ -155,20 +154,23 @@ export class Feedback extends Plugin<DragDropManager, FeedbackOptions> {
         x: initialCoordinates.x - relativeLeft,
         y: initialCoordinates.y - relativeTop,
       };
+
       const sizeDelta = {
         width:
-          (initialSize.width / initialFrameTransform.scaleX -
-            width / frameTransform.scaleX) *
+          (initialSize.width * initialFrameTransform.scaleX -
+            width * frameTransform.scaleX) *
           transformOrigin.x,
         height:
-          (initialSize.height / initialFrameTransform.scaleY -
-            height / frameTransform.scaleY) *
+          (initialSize.height * initialFrameTransform.scaleY -
+            height * frameTransform.scaleY) *
           transformOrigin.y,
       };
+
       const delta = {
-        x: coordinatesDelta.x / frameTransform.scaleX - sizeDelta.width,
-        y: coordinatesDelta.y / frameTransform.scaleY - sizeDelta.height,
+        x: coordinatesDelta.x / frameTransform.scaleX + sizeDelta.width,
+        y: coordinatesDelta.y / frameTransform.scaleY + sizeDelta.height,
```

Comment:
Had to rework this logic to account for regular size changes, too.

It seems to be mostly working, but keyboard sensor positions item a few px out, which has me suspicious